### PR TITLE
update log4cxx to 0.12.1

### PR DESCRIPTION
**Pull request by cc0der**
Specify library name and version:  **log4cxx/0.12.1**

Just a minor version bump for a library I am using. It fixes compilation for older compilers, that don't support C++17

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by cc0der** [comment]
What can I do to help this PR along? Is there anything I need to deliver?

**Comment by SSE4** [comment]
> What can I do to help this PR along? Is there anything I need to deliver?

have you already requested access, as the previous comment suggests?

**Comment by cc0der** [comment]
According to https://github.com/conan-io/conan-center-index/issues/4#issuecomment-948540126 I have been approved

**Comment by SSE4** [comment]
> According to [#4 (comment)](https://github.com/conan-io/conan-center-index/issues/4#issuecomment-948540126) I have been approved

yes, I can see you in the list of approved users. I have relaunched the CI.

File: recipes/log4cxx/config.yml [hunk 1]
```diff
@@ -1,3 +1,5 @@
 versions:
   "0.12.0":
     folder: all
+  "0.12.1":
+    folder: all
```
Comment (SSE4) [inline recipes/log4cxx/config.yml:5]:
```suggestion
    folder: all

```

**Pull request closed without merging**